bin/versions.ts

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+export async function getBbVersion(noirVersion: string) {
+  const url = `https://raw.githubusercontent.com/noir-lang/noir/v${noirVersion}/scripts/install_bb.sh`;
```

Comment:
Flagging that I'm not sure if / when @TomAFrench might plan to deprecate this script from Noir, which could break this noirenberg script here.

Reply:
Given how there is no good way to source Noir compatibility from within Barretenberg yet, don't think this should be blocking.

Reply:
Oh yeah, this script is purely an implementation detail of the noir repo CI and we don't provide any guarantees about its stability.